pandas-stubs/core/arrays/datetimelike.pyi

```diff
@@ -85,6 +82,5 @@ class DatetimeLikeArrayMixin(ExtensionOpsMixin, AttributesMixin, ExtensionArray)
     def mean(self, skipna: bool = ...): ...
 
 def validate_periods(periods): ...
-def validate_endpoints(closed): ...
```

Comment:
I think all 3 of the `validate_` methods can be deleted.  They don't seem to be documented.